Sources/SwifterSwift/SpriteKit/SKSpriteNodeExtensions.swift

```diff
@@ -0,0 +1,25 @@
+// SKSpriteNodeExtensions.swift - Copyright 2021 SwifterSwift
+
+#if canImport(SpriteKit)
+import SpriteKit
+
+// MARK: - Methods
+
+public extension SKSpriteNode {
+    /// SwifterSwift: SKSpriteNode sized with respect to aspect ratio.
+    ///
+    ///        node.aspectFill(to: CGSize(width: 300, height: 300)
+    ///
+    /// - Parameter fillSize: fill size to use for aspect ratio calculation.
+    func aspectFill(to fillSize: CGSize) {
+        if let texture = self.texture {
+            let horizontalRatio = fillSize.width / texture.size().width
```

Comment:
Let's be safe and check for `texture.size().width` being `0` to avoid division by zero crash, I think this would be unlikely but still nice to be safe =]

Reply:
I agree, will add. :)